src/compiler/crystal/compiler.cr

```diff
@@ -697,9 +701,11 @@ module Crystal
         end
 
         if @progress_tracker.stats?
+          name = result["name"].as_s
+          unit = units.find { |unit| unit.name == name }.not_nil!
```

Comment:
```suggestion
          unit = units.find!(&.name.== name)
```

Reply:
I'm fine doing `find!` instead but really don't like the equality block shorthand